infra/gravity/node_commands.go

```diff
@@ -113,6 +113,10 @@ type InstallParam struct {
 	InstallerURL string `json:"installer_url,omitempty"`
 	// OpsAdvertiseAddr is optional Ops Center advertise address to pass to the install command
 	OpsAdvertiseAddr string `json:"ops_advertise_addr,omitempty"`
+	// ServiceUID is an optional param for install's --service-uid flag: https://gravitational.com/gravity/docs/ver/7.x/pack/#service-user
```

Comment:
Nit: Really a nitpick but we tend not to shorten words in comments, so I'd say "parameter". Same below.

Reply:
Addressed in 786cc87.